src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Options/CollectionRuleOptionsExtensions.cs

```diff
@@ -168,6 +168,35 @@ public static CollectionRuleOptions AddLoadProfilerAction(this CollectionRuleOpt
                 });
         }
 
+        public static CollectionRuleOptions AddSetEnvironmentVariableAction(this CollectionRuleOptions options, string name, string value = null)
+        {
+            return options.AddAction(
+                 KnownCollectionRuleActions.SetEnvironmentVariable,
+                 callback: actionOptions =>
+                 {
+                     SetEnvironmentVariableOptions setEnvOpts = new()
+                     {
+                         Name = name,
+                         Value = value,
```

Comment:
nit: trailing comma

Reply:
In the absence of a style guide, I usually do this because when we add more options, it only causes a single line to be changed which can help git do merges easier